types/combined-stream/index.d.ts

```diff
@@ -1,19 +1,29 @@
 // Type definitions for combined-stream 1.0
 // Project: https://github.com/felixge/node-combined-stream
-// Definitions by: Felix Geisendörfer <https://github.com/felixge>, Tomek Łaziuk <https://github.com/tlaziuk>
+// Definitions by: Felix Geisendörfer <https://github.com/felixge>, Tomek Łaziuk <https://github.com/tlaziuk>, Kon Pik <https://github.com/konpikwastaken>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 /// <reference types="node" />
 
-import { Stream } from "stream";
+import { Stream } from 'stream';
+import { ReadStream } from 'fs';
```

Comment:
where is this import used?

Reply:
Oops, not sure how that got added. Removing.